test_calculator.py

```diff
@@ -8,4 +7,27 @@ def test_calculator_add():
     assert calculator.calculator_add(0, 0) == 0
     assert calculator.calculator_add(0, 1) == 1
     assert calculator.calculator_add(0, -5) == -5
-
+def test_calculator_subtract():
```

Comment:
add a space between the last line of the last test and first line of the next test